include/libtorrent/session.hpp

```diff
@@ -149,6 +149,8 @@ namespace libtorrent
 		libtorrent::dht_settings dht_settings;
 
 		dht::dht_storage_constructor_type dht_storage_constructor;
+
+		bdecode_node state;
```

Comment:
I think there is a member to hold this in ``session_impl`` already.. isn't there?